src/Symfony/Component/Mailer/Bridge/Infobip/Tests/Transport/InfobipApiTransportTest.php

```diff
@@ -39,10 +39,7 @@ public function request(string $method, string $url, array $options = []): Respo
                 // The only purpose of this method override is to record the request body as a string
                 // It's impossible to get the generated body when using a generator as a body
                 if (isset($options['body']) && $options['body'] instanceof \Generator) {
-                    $body = '';
-                    foreach ($options['body'] as $data) {
-                        $body .= $data;
-                    }
+                    $body = implode('', $options['body']);
```

Comment:
This won't work. `implode()` is made for arrays, `$options['body']` is a generator (see the line above your change).

Reply:
Sorry! The automatic tool totally fooled me on this. I reverted this and all the other occurrences. Thanks.